.github/workflows/develop.yml

```diff
@@ -29,5 +29,5 @@ jobs:
           touch .env
           echo "${{ secrets.DEV_ENV_FILE }}" > .env
 
-      - name: Start PM2 process
-        run: pm2 start server.js --name "ReminderBot_dev" --time
+      - name: Reload PM2 process
```

Comment:
if u wondering why here reload instead start cause it's already exist in pm2, else it gives an error, after this merge production.yml also becomes reload